packages/context/src/inject.ts

```diff
@@ -124,7 +124,7 @@ export function inject(
   }
   return function markParameterOrPropertyAsInjected(
     target: Object,
-    member: string,
+    member: string | undefined,
```

Comment:
The `ParameterDecorator` type from TypeScript is declared as follows:
```ts
declare type ParameterDecorator = (target: Object, propertyKey: string | symbol, parameterIndex: number) => void;
```

Can we use `''` for constructor parameters?

Reply:
Interestingly, at runtime, the `propertyKey` is `undefined` in constructor cases.

Reply:
> Interestingly, at runtime, the `propertyKey` is `undefined` in constructor cases.

That's the weird thing!